core/core.ts

```diff
@@ -576,38 +576,18 @@ export function fileGenerator(): FileGenerator {
     return result;
   }
 
-  const INDENT = '  ';
-
-  function textLines(indent: string, prefix: string, text: string): string[] {
-    return [indent + prefix + text];
-  }
-
-  function textValue(t: string):  hcl2.ExprTerm {
-    const imatch =  t.match(/^"\${([0-9A-Za-z._]+)}\"$/);
-    if (imatch != null) {
-      return hcl2.getVariable(imatch[1]);
-    } else if (t.startsWith('"') && t.endsWith('"')) {
-      return hcl2.stringLit(t.substr(1, t.length-2));
-    } else {
-      return hcl2.stringLit(t);
-    }
-  }
-
-
   function generateMetadata(resource: ResourceDetails): hcl2.BodyItem[] {
     const items: hcl2.BodyItem[] = [];
 
-    {
     const lifecycle_items: hcl2.BodyItem[] = [];
```

Comment:
I prefer it as it was, where `lifecycle_items` was only scoped as required, rather scoped to the end of the function?

Reply:
Will fix it

Reply:
I don't really care - I'm just pointing out it was written as is intentionally.